database/src/bin/sqlite-to-postgres.rs

```diff
@@ -56,6 +60,10 @@ impl Table for Artifact {
         "id, name, date, type"
     }
 
+    fn postgres_attributes() -> &'static str {
+        "id, name, date, type"
```

Comment:
Alternatively, we could use a macro to get the field names of the row struct instead of hardcoding them.